### DISCUSSION TITLE
how can set on pages are visible to all or only for internal purposes?

### DISCUSSION THREAD

**Discussion by poltrian** [requarks/wiki#5293] [Help / Questions]
I want to use Wiki JS for internal documentation but also publish pages for everyone, how can I set this up?

**Comment by NGPixel** [comment]
https://docs.requarks.io/groups

**Comment by poltrian** [comment]
Sorry but can not find a anything like "not visible with no account". Its very confused.

**Reply by ghost** [reply]
> Sorry but can not find a anything like "not visible with no account". Its very confused.

I figured it out.  I needed this feature too, and you can do it!

Go to Groups -> Guest, click on the Page Rules tab

Add a new rule. Set it to DENY (click the green checkmark to change it to red).  Deny "read page", "read assets", "read comments".
Choose "Page starts with.." and `/internal-stuff/`

![image](https://user-images.githubusercontent.com/89231731/168490147-a9203441-3ba8-4aac-91c8-95330daae66a.png)

Now the guest users will not be able to read anything inside your `/internal-stuff/` folder or any subfolders/sub-pages.

**Reply by poltrian** [reply]
Thank you very much, now i understand what they mean.
